### PR TITLE
API for getting messages

### PR DESCRIPTION
**Pull request by szczeles**
Thanks for your project, it helped me a lot!

I'm adding here a new API for getting messages (envelope icon marked as "Wiadomości" on the top bar).

### CODE REVIEW

File: librus/__init__.py [hunk 1]
```diff
@@ -172,6 +172,28 @@ def schedule(self):
...
+            message = Message(
+                sender=cells[2].text,
+                subject=cells[3].text,
+                sent_at=datetime.datetime.strptime(cells[4].text, '%Y-%m-%d %H:%M:%S')
+            )
+            if get_content:
+                href = cells[3].find('a')[0].attrs['href']
+                url = 'https://synergia.librus.pl' + href
+                content = self._html_session.get(url=url).html.find('.container-message-content')[0]
+                message.set_content(content.text)
```
Comment (findepi) [inline librus/__init__.py]: is there any particular advantage of `set_content` vs `content = ` ?

like 

```suggestion
                message.content = content.text
```

?

**Review by findepi — APPROVED** [review]
@szczeles thanks!

looks great, one comment

(i admit i didn't test it  😉 )

File: librus/__init__.py (hunk 1 above)
Comment (szczeles) [reply librus/__init__.py]: No reason, I like your way of keeping it simple!

**Comment by szczeles** [comment]
No worries, I'm constantly testing it via https://github.com/szczeles/toolbelt/tree/master/librus-messages-notifier ;-)

File: librus/__init__.py [hunk 2]
```diff
@@ -178,7 +178,7 @@ def list_messages(self, get_content=False):
         """
         response = self._html_session.get(url='https://synergia.librus.pl/wiadomosci')
 
-        for row in response.html.find('.line0') + response.html.find('.line1'):
+        for row in response.html.find('.stretch > tbody > tr'):
```
Comment (findepi) [inline librus/__init__.py:181]: intentional? (the commit message says `Setting message content directly`, so i want to make sure)

Comment (szczeles) [reply librus/__init__.py:181]: Yep, it is all right. Just a quick fix to get the messages in order they appear in the html.

**Comment by findepi** [comment]
Merged, thanks!

**Comment by findepi** [comment]
Released as 0.0.10

**Pull request merged**